packages/api/src/utils/toSafeNumber.ts

```diff
@@ -0,0 +1,12 @@
+import { transform } from 'lodash';
+import BigNumber from 'bignumber.js';
+
+export default function toSafeNumber(object: Object): Object {
```

Comment:
Would you mind adding some comments explaining what this function does and why it's important?

Reply:
I also find the name to be a bit confusing since I see toSafeNumber() being applied to objects that aren't immediately obvious as being (or containing) numeric (or having numeric properties).

Reply:
I added this function because json-bigint has a bug https://github.com/sidorares/json-bigint/issues/63 and I would like to have all numbers <= Number.MAX_SAFE_INTEGER as the standard type. You are not able to use BigNumber directly as React value and it will throw an exception when you try to render it.